server/util/walkDir.js

```diff
@@ -0,0 +1,47 @@
+import path from 'path';
+import fs from 'fs-extra';
+
+/**
+ * Walks the directory, and returns all nested files of designated file name.
+ * Only traverses one layer.
+ * @example walkJsonDir('test', 'test.json') will find 'test/1/test.json'
```

Comment:
Have you considered using glob patterns instead? It would reduce a lot of custom directory walking code. I would recommend https://github.com/isaacs/node-glob.